src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,18 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:
+        return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+
+    def publish_manifest(self):
```

Comment:
maybe this method should remain commented out for now

Reply:
Agree, please leave this commented out for now

Reply:
which should be commented? the `meta` method or the `publish_manifest` method?

Reply:
the whole `publish_manifest` method, just add tests for `project.meta` with different configs